pkg/config/config.go

```diff
@@ -543,6 +543,8 @@ type ImageConfig struct {
 	// ImageVolumes controls how volumes specified in image config are handled
 	ImageVolumes ImageVolumesType `toml:"image_volumes"`
 	// Registries holds a list of registries used to pull unqualified images
+	// Deprecated: Support for this option has been dropped, and it has no effect.
+	// Please refer to containers-registries.conf(5) for configuring unqualified-search registries.
 	Registries []string `toml:"registries"`
```

Comment:
I wonder if we should just be done with it and remove it already? It's been a while since 2021...

@cri-o/cri-o-maintainers, thoughts on removing this completely?

Reply:
Have we ever deprecated this? If not, then we have to according to https://github.com/cri-o/cri-o/blob/main/deprecating_process.md

Reply:
Would this stand in for deprecation?

- https://github.com/cri-o/cri-o/blob/main/pkg/config/config.go#L765-L770

This has been defunct for about two years now.

Reply:
Somehow, yes.

Reply:
We can:

1. Remove it completely, given that it has been defunct for a while, with a relevant log line being produced each time this option has been enabled/used.

2. Improve messaging about this option being detected in command-line use and documentation, and perhaps even make the log line more explicit.

@cri-o/cri-o-maintainers, thoughts?